data/Pirralho_2014/metadata.yml

```diff
@@ -23,10 +23,10 @@ contributors:
 dataset:
   data_is_long_format: no
   custom_R_code:           '
-    data %>% 
+    data %>%
       mutate(
         site_name = "University of Lisbon campus fields",
-        across(2:15, str_replace, "±.*", ""),
+        across(2:15, str_replace, "(\U00B1).*", ""),
```

Comment:
I'm curious if this will still be required together with the changes you made on traits.build - before we merge in this branch, we should try running the old version of this together with the new code on traits.build and see what happens.

Reply:
I think I already tested it and this change isn't needed with the new traits.build code! But for this release we need it.